static/app/stores/configStore.spec.tsx

```diff
@@ -19,12 +29,7 @@ describe('ConfigStore', () => {
   });
 
   it('should have customerDomain', () => {
-    const customerDomain = ConfigStore.get('customerDomain');
-    expect(customerDomain).toEqual({
-      subdomain: 'foobar',
-      organizationUrl: 'https://foobar.sentry.io',
-      sentryUrl: 'https://sentry.io',
-    });
+    expect(ConfigStore.get('customerDomain')).toEqual(null);
```

Comment:
This is kind of a low value test as it mostly validates the default data loaded into `ConfigStore`.